common/LongPathHandler.go

```diff
@@ -0,0 +1,46 @@
+package common
+
+import (
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+//https://docs.microsoft.com/en-us/windows/desktop/FileIO/naming-a-file#maximum-path-length-limitation
+//Because windows doesn't (by default) support strings above 260 characters,
+//we need to provide a special prefix to tell it to support paths up to 32,767 characters.
+
+//Furthermore, we don't use the built-in long path support in the Go SDK commit 231aa9d6d7
+//because it fails to support long UNC paths. As a result, we opt to wrap things such as filepath.Glob()
+//to safely use them with long UNC paths.
+
+//ToExtendedPath converts short paths to an extended path.
+func ToExtendedPath(short string) string {
+	short, err := filepath.Abs(short)
+	PanicIfErr(err) //TODO: Handle errors better?
+
+	if os.PathSeparator == '\\' {
```

Comment:
> os.PathSeparator [](start = 3, length = 17)

perhaps test on WSL, to see how it behaves

Reply:
I'm on WSL2 but my home machine cooperated on GA WSL.

Reply:
I'm changing these two to runtime.GOOS == "windows" just to be stricter.